[Repository] Add .github folder with contributing file (#3362)
lukesampson/scoop: .github/ISSUE_TEMPLATE/enhancement.md

---
name: Enhancement

> about: Some feature could be better.

Reviewer: ```suggestion
about: Some feature, idea or enhancement to improve Scoop.
```

labels: enhancement
---

> <!-- Describe your idea how to make some feature better. -->

Reviewer: ```suggestion
<!-- Describe your idea about how to improve Scoop. -->
```
